spec/injectable/dependencies_graph_spec.rb

```diff
@@ -1,4 +1,4 @@
-describe Injectable::DependenciesGraph, '#resolve' do
+RSpec.describe Injectable::DependenciesGraph, '#resolve' do
```

Comment:
That is not necessary, if you need it then you have RSpec misconfigured

Reply:
Also, that `'#resolve'` tag should not be there or in any of the others. I guess you need a new `describe` block

Reply:
To configure `RSpec` to not need to do `RSpec.describe` you just remove this line: https://github.com/rubiconmd/injectable/blob/121944c53125cb98f35a68e4e984603b36b34a8d/spec/spec_helper.rb#L9

Reply:
Thanks!